### PR TITLE
Add  file manager configs

### PR DESCRIPTION
**Pull request by vincent-karuri**

### CODE REVIEW

File: roles/opensrp/templates/opensrp_local_checkout_path/configs/assets/config/opensrp.properties.j2 [hunk 1]
```diff
@@ -116,3 +116,14 @@ opensrp.cors.max.age={{ opensrp_cors_max_age }}
...
+
+# S3 configuration (should be populated for deployments using S3 multimedia storage)
+aws_access_key_id=
+aws_secret_access_key=
+aws_region=
+aws_bucket=
+aws_key_folder=
+
+# multimedia file manager (is file system by default)
+multimedia_file_manager=FileSystemMultimediaFileManager
```
Comment (githengi) [inline roles/opensrp/templates/opensrp_local_checkout_path/configs/assets/config/opensrp.properties.j2]: Add variables to each of these configs
 
for the file manager default the variable on the OpenSRP role

**Review by githengi — CHANGES_REQUESTED** [review]

File: roles/opensrp/templates/opensrp_local_checkout_path/configs/assets/config/opensrp.properties.j2 (hunk 1 above)
Comment (vincent-karuri) [reply roles/opensrp/templates/opensrp_local_checkout_path/configs/assets/config/opensrp.properties.j2]: Yea, this was just a quick fix to enable deployment.

Working on more comprehensive issue [here](https://github.com/OpenSRP/playbooks/issues/154).

Comment (vincent-karuri) [reply roles/opensrp/templates/opensrp_local_checkout_path/configs/assets/config/opensrp.properties.j2]: Done.

File: roles/opensrp/defaults/main.yml [hunk 2]
```diff
@@ -117,7 +117,17 @@ opensrp_cors_allowed_source: ""
 opensrp_cors_max_age: "60"   
 
 #Team module settings
-opensrp_use_opensrp_team_module: false  
+opensrp_use_opensrp_team_module: false
+
+# S3 configuration (should be populated for deployments using S3 multimedia storage)
+aws_access_key_id: ""
```
Comment (jasonrogena) [inline roles/opensrp/defaults/main.yml]: @vincent-karuri please prefix all these variables with `opensrp_`

Comment (vincent-karuri) [reply roles/opensrp/defaults/main.yml]: @jasonrogena That's possible but it would require changing the code that depends on it in opensrp server

Comment (vincent-karuri) [reply roles/opensrp/defaults/main.yml]: Can this change be part of this issue #154?

This was just meant to be a quick fix to allow deployments

Comment (jasonrogena) [reply roles/opensrp/defaults/main.yml]: The Ansible variables, and not the ones defined in the .properties file.

Comment (vincent-karuri) [reply roles/opensrp/defaults/main.yml]: Done.

**Review by githengi — APPROVED** [review]

**Pull request merged**